### ISSUE TITLE
Create a game.

### ISSUE THREAD

**Issue by NatasaPeic** [ga-wdi-boston/game-project#211]
What are we passing when creating a new game? An ID?

**Comment by gaand** [comment]
What does the game api documentation say?

**Comment by NatasaPeic** [comment]
An ID, cells, player_x and player_x's id and email.

**Comment by gaand** [comment]
That's what is says is in the response.

From https://github.com/ga-wdi-boston/game-project-api#create

The `create` action expects a _POST_ with an empty body (e.g `''` or `'{}'` if JSON). If the request is successful, the response will have an HTTP Status of 201 Created, and the body will contain JSON of the created game with `player_x` set to the user calling `create`

**Comment by laurenfazah** [comment]
@NatasaPeic were you able to solve this? If so could you close with a summary? If not, please ask further questions

**Comment by NatasaPeic** [comment]
No. I haven't solved it. I captured data when a user signed up, but not sure where to go from there. Thanks.

**Comment by laurenfazah** [comment]
Like Antony said, to create a game, all is needed is a `POST` request to `/games`.

The docs also say:

> All games action requests must include a valid HTTP header `Authorization: Token token=<token>` or they will be rejected with a status of 401 Unauthorized.

**Comment by laurenfazah** [comment]
@NatasaPeic was it solved? Could you summarize your solution if so? Thanks!

**Comment by NatasaPeic** [comment]
I'm working on it right now. I closed this issue because answer was provided. Thank you.

**Comment by laurenfazah** [comment]
Okay. You're welcome to leave issues open until they are solved. Feel free to reopen if the issue continues.
